test_data_api.py

```diff
@@ -82,6 +86,22 @@ def test_ecmwf_tc_bufr(self):
         fcast = TCForecast.fetch_bufr_ftp()
         [f.close() for f in fcast]
 
+    def test_icon_forecast_download(self):
+        """Test availability of DWD icon forecast."""
+        run_date = dt.datetime.today().replace(hour=0,
+                                               minute=0,
+                                               second=0,
+                                               microsecond=0)
```

Comment:
given the lag between the ICON model run and its availability on the DWD site ... will this test fail between midnight and ~3:30am?

Reply:
ughh and in time zones that are ahead of germany?? what time zone is jenkins in?

Reply:
I was able to adapt it, so the test can run at any time.